### PR TITLE
fix(react-components): Toolbar styling and missing buttons toggle

### PR DESCRIPTION
**Pull request by pramodcog**
#### Type of change

Bug 

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/

## Description :pencil:
The toolbar is corrected to match Figma styling
Add toggle action for all buttons 

## How has this been tested? :mag:

In storybook

## Test instructions :information_source:

`cd react-components && yarn && yarn build && yarn storybook`
Select `Toolbar` example

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [x] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [ ] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [ ] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

**Comment by pramodcog** [comment]
> image
> 
> Is incorrect vs. Figma styling
> 
> image

Confirmed with Kerem on the styling, he is good with the current styling as it is the default css property set in the `Menu.Item`

File: react-components/src/components/RevealToolbar/LayersButton.tsx [hunk 1]
```diff
@@ -40,6 +40,25 @@ export const LayersButton = (): ReactElement => {
     setVisible((prevState) => !prevState);
   };
 
+  const outsideClick = (ref: RefObject<HTMLElement | null>): void => {
```
Comment (christjt) [inline react-components/src/components/RevealToolbar/LayersButton.tsx:43]: This is a hook. The convention use to prefix the name with `use`

File: react-components/src/components/RevealToolbar/LayersButton.tsx [hunk 2]
```diff
@@ -40,6 +40,25 @@ export const LayersButton = (): ReactElement => {
...
+    useEffect(() => {
+      const handleClick = (event: MouseEvent): void => {
+        if (ref.current !== null && !ref.current.contains(event.target as Node)) {
```
Comment (christjt) [inline react-components/src/components/RevealToolbar/LayersButton.tsx:46]: What happens if event.target isn't a Node here?

File: react-components/src/components/RevealToolbar/RevealToolbar.tsx [hunk 3]
```diff
@@ -61,20 +65,17 @@ const RevealToolbarContainer = ({
   toolBarContent,
   ...restProps
 }: RevealToolbarProps & { toolBarContent?: JSX.Element }): ReactElement => {
-  if (restProps.className === undefined && restProps.style === undefined) {
-    restProps = { ...restProps, ...defaultStyle };
-  }
   return (
     <I18nWrapper translations={translations} addNamespace="reveal-react-components">
-      <ToolBar {...restProps}>
+      <StyledToolBar {...restProps}>
```
Comment (christjt) [inline react-components/src/components/RevealToolbar/RevealToolbar.tsx:70]: Seem to be getting a type error here.

**Review by christjt — CHANGES_REQUESTED** [review]
Looks good, a few comments.

**Review by christjt — CHANGES_REQUESTED** [review]
React spews out an error on initial click on of the layers button.
image

**Comment by pramodcog** [comment]
> React spews out an error on initial click on of the layers button. image

This is coming from `cogs.js`, I don't have a workaround currently

**Review by christjt — APPROVED** [review]
👍

**Pull request merged**